routes/conversations.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const Router = require('express').Router;
+const Conversation = require('../models/Conversation');
+const Message = require('../models/Message');
+const sgMail = require('@sendgrid/mail');
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+const router = Router();
+
+
+router.post('/', (req, res) => {
+
+  let conversation = new Conversation({
+    from_email: req.body.from_email,
+    to_email: req.body.to_email
+  });
+
+  conversation.save((err) => {
```

Comment:
This POST route should only respond with json, not page redirects.

If the resource already exists, this route should report back a 400 since the user's goal was not achieved (the goal being to CREATE a new resource)

Reply:
ok. It does make sense because this route on has one job. So i see why redirect is not ok to use here.